.circleci/config.yml

```diff
@@ -76,12 +76,15 @@ commands:
   build-maven-java:
     description: Build maven-java
     steps:
+      - run: 
+          name: publish m2
+          command: sbt codegenJava/publishM2
       - run:
           name: "Build maven-java"
           command: |
             cd maven-java
             echo "Running maven-java with SDK version: '$SDK_VERSION'"
-            mvn versions:set -DnewVersion=$SDK_VERSION
+            mvn versions:set -DnewVersion=$SDK_VERSION -Dakkaserverless-codegen.version=$SDK_VERSION
```

Comment:
Interested to see how this will work out. 

Before building the plugin we need to build the codegen and we must change the codegen version in the pom file with `-Dakkaserverless-codegen.version`. 

I'm curious to see what will be the release pom. Will that have the original `akkaserverless-codegen.version` value? If so, the build will have to effectively modify the file before publishing.